Package:     chrony
Subject:     fix small typo in manpage
Author:      Joachim Wiedorn <ad_debian at joonet.de>
Forwarded:   yes, per mail at 08.08.2014
Last-Update: 2014-08-08
---

diff -urN s00/chronyd.8.in s01/chronyd.8.in
--- s00/chronyd.8.in	2014-07-01 14:50:19.000000000 +0200
+++ s01/chronyd.8.in	2014-08-07 14:59:58.585668759 +0200
@@ -76,7 +76,7 @@
 .B \-R
 When this option is used, the \fIinitstepslew\fR directive and the
 \fImakestep\fR directive used with a positive limit will be ignored. This
-option is useful when restarting \fBchronyd\fR and can be used in conjuction
+option is useful when restarting \fBchronyd\fR and can be used in conjunction
 with the \fB-r\fR option.
 .TP
 .B \-s
