Package:     chrony
Subject:     add version header file
Author:      Joachim Wiedorn <ad_debian at joonet.de>
Forwarded:   no
Last-Update: 2013-11-22
---

diff -urN s01/version.h s02/version.h
--- s01/version.h	1970-01-01 01:00:00.000000000 +0100
+++ s02/version.h	2013-11-22 14:48:00.000000000 +0100
@@ -0,0 +1,4 @@
+#ifndef VERSION_H
+#define VERSION_H 1
+#define PROGRAM_VERSION_STRING "1.29"
+#endif /* VERSION_H */
